yang/ac-svc-tree-RR-revision.txt

```diff
@@ -176,19 +176,43 @@ module: ietf-ac-svc
 // We also provide a placeholder to be populated with adequate parameters. The logic of the policy is 
 // Do we want authentication here ?
 //
+// + Adding local-address (i.e PE address), in case we have a loopback on the PE that must be learnt by the customer orchestrator ??
```

Comment:
Good point. The address should be a read-only then.

Reply:
100% agree !

Reply:
This is now reflected in the YANG module.